lib/components/ms-toggle/types.ts

```diff
@@ -4,3 +4,8 @@ export enum DisplayState {
   List = 0,
   Grid = 1,
 }
+
+export enum Answer {
```

Comment:
`MsModal` has a type `Answer` already, https://github.com/Scille/megashark-lib/blob/cae49fd4774b08b8064a35502133f90cb3484f33/lib/components/ms-modal/types.ts#L7

It should probably be moved to `ms-types.ts` instead. Don't forget to re-export it from MsModal, else we will have a lot of missing imports.